packages/@react-spectrum/picker/test/Picker.test.js

```diff
@@ -520,29 +525,24 @@ describe('Picker', function () {
       );
 
       expect(queryByRole('listbox')).toBeNull();
+      let selectTester = testUtilUser.createTester('SelectTester', {root: getByRole('button')});
+      let picker = selectTester.trigger;
+      await selectTester.open();
 
-      let picker = getByRole('button');
-      await user.click(picker);
-      act(() => jest.runAllTimers());
-
-      let listbox = getByRole('listbox');
+      let listbox = selectTester.listbox;
       expect(listbox).toBeVisible();
       expect(onOpenChange).toBeCalledTimes(1);
       expect(onOpenChange).toHaveBeenCalledWith(true);
       expect(picker).toHaveAttribute('aria-expanded', 'true');
       expect(picker).toHaveAttribute('aria-controls', listbox.id);
 
-      fireEvent.keyDown(listbox, {key: 'Escape'});
-      act(() => jest.runAllTimers());
+      await selectTester.close();
 
-      expect(listbox).not.toBeInTheDocument();
```

Comment:
Don't we have a falsy check like, `expect(selectTester.listbox).toBeFalsy();` for this or did you determine it's redundant?

Reply:
The utility actually checks that the listbox should no longer be in the DOM for you when calling `close`